### PR TITLE
Update file complete

### PR DESCRIPTION
**Pull request by tytynono66**
Please update Files UWP from the Microsoft Store, the french version is so bad.

### CODE REVIEW

**Comment by yaichenbaum** [comment]
Can you please target the `develop` branch?

**Comment by xPoppyx** [comment]
don't forget to change the state from "new" to "translated"

**Comment by tytynono66** [comment]
How ? i don’t understand


Provenance : Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows 10

De : Yair Aichenbaum<mailto:notifications@github.com>
Envoyé le :vendredi 24 avril 2020 20:33
À : duke7553/files-uwp<mailto:files-uwp@noreply.github.com>
Cc : tytynono66<mailto:tytynono@msn.com>; Author<mailto:author@noreply.github.com>
Objet :Re: [duke7553/files-uwp] Update file complete (#680)


Can you please target the develop branch?

File: Files/MultilingualResources/Files.fr-FR.xlf [hunk 1]
```diff
@@ -414,209 +414,205 @@
...
+          <target state="new">AVERTISSEMENT: DES FONCTIONS EXPÉRIMENTALES À VENIR!</target>
         </trans-unit>
         <trans-unit id="SettingsNavExperimental.Content" translate="yes" xml:space="preserve">
           <source>Experimental</source>
-          <target state="new">Experimental</target>
+          <target state="new">Expérimental</target>
         </trans-unit>
         <trans-unit id="SettingsExperimentalDoubleTapToRenameFiles.Text" translate="yes" xml:space="preserve">
           <source>Double tap to rename files</source>
-          <target state="new">Double tap to rename files</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.fr-FR.xlf:613]: It looks like you removed some text here, can you add them back in?

**Comment by tytynono66** [comment]
I remove Nothing

[cid:image003.jpg@01D61A7B.AA1C9DB0]

I don’t understand what you want.
I attach the xlf file
I just wanted to have this great application in french.
tytynono



De : Yair Aichenbaum<mailto:notifications@github.com>
Envoyé le :vendredi 24 avril 2020 20:45
À : duke7553/files-uwp<mailto:files-uwp@noreply.github.com>
Cc : tytynono66<mailto:tytynono@msn.com>; Author<mailto:author@noreply.github.com>
Objet :Re: [duke7553/files-uwp] Update file complete (#680)


@yaichenbaum requested changes on this pull request.

Pease address the issues mentioned in my review.


In Files/MultilingualResources/Files.fr-FR.xlf<https://github.com/duke7553/files-uwp/pull/680#discussion_r414788060>:

>          </trans-unit>

         <trans-unit id="SettingsExperimentalDoubleTapToRenameFiles.Text" translate="yes" xml:space="preserve">

           <source>Double tap to rename files</source>

-          <target state="new">Double tap to rename files</target>

It looks like you removed some text here, can you add them back in?

**Comment by xPoppyx** [comment]
@tytynono66  you accidentally deleted these, you must add them again:
Anotación 2020-04-24 151137

**Pull request closed without merging**